### PR TITLE
feature/ISDK-1020: Update agora lib

### PR DESCRIPTION
**Pull request by GraceFirework**
## Objective :dart:

Update Agora to resolve an issue where livestream audio would continue playing even after moving away from the tab.

  * Ticket: feature/ISDK-1020
  
## Changes :boom:

**Changes that you made in this Pull Request.**

## Type of Pull Request 

 - [ ] Feature
 - [ ] Bug Fix
 - [ ] Enhancement
 - [ ] Maintenance, Code Refactor

## Dependent PRs :clipboard:

**List of other PRs this PR depends on, if any**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,12 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.4.1]
+
+### Fixed
+
+  - Resolved an issue where livestream audio would continue playing even after moving away from the tab.
```
Comment (GraceFirework) [inline CHANGELOG.md:10]: do we usually give this much detail for supporting libs? Or do I just mention bug fix?

**Comment by GraceFirework** [comment]
@spookyd  when you have some time can you review this PR please

**Review by spookyd — APPROVED** [review]
LGTM

**Review by jlebajan — COMMENTED** [review]
Shall we also update the podspec file?

**Review by jlebajan — APPROVED** [review]
LGTM.

**Pull request merged**